crates/nu-command/tests/commands/mut_.rs

```diff
@@ -148,3 +148,14 @@ fn def_should_not_mutate_mut() {
     assert!(actual.err.contains("capture of mutable variable"));
     assert!(!actual.status.success())
 }
+
+#[test]
+fn mut_with_bad_syntax_raises_parse_error() {
```

Comment:
Could you make the name of this test a little clearer? `assign_to_non_mut_variable_raises_parse_error`

Reply:
Sure, I changed the name